docs/examples/stocks-portfolio.md

```diff
@@ -1,20 +1,20 @@
 # Stocks Portfolio
 
-We will create a skill that will allow opsdroid to return info of specific stocks. It will also be able to store a portfolio of stocks and return each with the current market price and info of it.
+We will create a skill that will allow the opsdroid to return info on specific stocks. It will also be able to store a portfolio of stocks and return each with the current market price and information about them.
```

Comment:
```suggestion
We will create a skill that will allow opsdroid to return info on specific stocks. It will also be able to store a portfolio of stocks and return each with the current market price and information about them.
```